### PR TITLE
Create a photo gallery page

### PR DESCRIPTION
**Pull request by AmilaIndika789**
I have created a photo gallery page which could include albums of various events and sports held in the college. Added a side bar and quotes of several famous sportsmen.  At the bottom there is a button named find more, thought of using it to link to photographers page. Any suggestions or comments?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "react-dom": "^16.6.3"
   },
   "devDependencies": {
+    "@zeit/next-css": "^1.0.1",
```
Comment (shyaman) [inline package.json]: Is this a dev dependency?

File: pages/photos.js [hunk 2]
```diff
@@ -0,0 +1,968 @@
...
+                        <li><a href="#">Photos</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </nav>
+
+                <div className="fh5co-footer">
+                    <p>
+                        <small>&copy; 2018 PeraSports, UOP. All Rights Reserved.
+                            <span>Designed by <a href="http://freehtml5.co/" target="_blank">Engineering Faculty, UOP</a> </span>
```
Comment (shyaman) [inline pages/photos.js]: let's add this link here [http://www.ce.pdn.ac.lk/](http://www.ce.pdn.ac.lk/)

File: pages/photos.js [hunk 3]
```diff
@@ -0,0 +1,968 @@
...
+                            <span>Images: <a href="http://pexels.com/" target="_blank">Pera Beats</a></span>
```
Comment (shyaman) [inline pages/photos.js]: We can add perabeats fb page

File: package.json (hunk 1 above)
Comment (AmilaIndika789) [reply package.json]: @shyaman No I tried to remove, I haven't used it actually, just added during testing to add a feature. When I tried to remove it, there was a travis build failure. Check if it is possible to remove it without build failure

File: pages/photos.js (hunk 2 above)
Comment (AmilaIndika789) [reply pages/photos.js]: @shyaman Added the link, resolving the conversation

File: pages/photos.js (hunk 3 above)
Comment (AmilaIndika789) [reply pages/photos.js]: @shyaman Added the link, resolving the conversation

File: package.json (hunk 1 above)
Comment (AmilaIndika789) [reply package.json]: @shyaman Seems like issue is resolved, removed the unwanted @zeit/next-css package without any build failure. Please check and resolve if problem still exists.

**Review by shyaman — APPROVED** [review]

**Pull request merged**